lib/common/api/native-image.js

```diff
@@ -1 +1,15 @@
-module.exports = process.atomBinding('native_image')
+const {deprecate} = require('electron')
+
+const nativeImage = process.atomBinding('native_image')
+
+nativeImage.toPng = function () {
+  deprecate.warn('nativeImage.toPng()', 'nativeImage.toPNG()')
```

Comment:
I saw on other files that you wrapped deprecations in a conditional: `if (!process.noDeprecations)` Should that be done here too?